packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -1560,4 +1578,66 @@ const importTable = async (tableDef: CustomDDB.Input): Promise<AWSCDKAsyncCustom
   return result;
 };
 
+export const getImportedTableComparisonProperties = (importedTable: TableDescription): TableComparisonProperties => {
```

Comment:
1. Does this need to be exported? It looks like it's used above, but I can't tell that it's used elsewhere (maybe it's exported for tests though)
2. This file is already pretty long & unwieldy -- can we refactor the table comparison logic and this new helper into a separate set of files to make it easier to figure out what the heck is going on in here? I'd also suggest you create field-specific helpers for any field that's not a straight up `ComparisonField: importedTable.Field` assignment, again to improve readability. The current structure makes it pretty hard to parse.

Reply:
1. Exported for tests
2. Would you prefer this in a follow up or done in this PR?